pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -887,31 +887,48 @@ func (oe *operationExecutor) generateVerifyControllerAttachedVolumeFunc(
 				volumeToMount.Pod.UID)
 		}
 
+		// This will write to node status that we're about to start using the volume.
+		// If we don't need the given volume, there might be a delay of upto 10s
+		// before we proclaim that we don't need it, because we *don't* synchronously
+		// delete the same volume from the node status map.
+		// TODO: consider fixing this post 1.3.
+		addVolumeNodeErr := actualStateOfWorld.MarkVolumeAsAttached(
```

Comment:
I did it this way vs a new operation in the operation executor because 1. I think this is the smallest change in terms of LOC, 2. I don't think we want to every say a volume is in use without syncing it with apiserver.